docs/articles/baseplatform/creating-a-new-layer.rst

```diff
@@ -182,3 +182,18 @@ won't need to modify all the recipes.
 
 .. [#bbmanual] http://www.yoctoproject.org/docs/latest/bitbake-user-manual/bitbake-user-manual.html
 .. [#templateconf] http://www.yoctoproject.org/docs/latest/dev-manual/dev-manual.html#creating-a-custom-template-configuration-directory
+
+Optionally depending on other layers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In some cases a layer might have recipes optional dependencies on other layers. For instance, one
+might want to append on a recipe for a Qt application which will only be built for some qt specific
+image. Adding the recipe directly to the layer will cause bitbake to generate errors about dangling
+appends when building non-Qt images. To mitigate this, create a "layers" directory containing
```

Comment:
Perhaps one could elaborate more when specifying non-qt image, to say that such an image wouldn't include the qt layers - hence creating dangling appends.

Reply:
fixed